exodus-installer.sh

```diff
@@ -52,19 +57,46 @@ exodus_download() {
 }
 
 
+# Download and check the exodus package to verify
+# SHA hash
+exodus_verify_hashes() {
+  #
+  # If JP's key doesn't exist...
+  if ! gpg -k | grep -q "JP Richardson" ; then 
```

Comment:
We should check by KeyID, not name...
Right way: `gpg --list-public-keys --with-colons --fixed-list-mode | grep -q '9D959455325B781A'`

`--with-colons` docs: https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=blob;f=doc/DETAILS

ping @kklash (same issue for verifyLinux.sh)

Reply:
Agreed. That did bother me a bit, but I figured since that was the way it was done in the other script, I'd go with it first version.